autoscale/src/test/java/com/sequenceiq/periscope/endpointtests/DistroXAutoScaleClusterV1EndpointTest.java

```diff
@@ -385,6 +393,59 @@ public void testUpdateAutoscaleConfigByClusterCrnForMultipleLoadAlertsAndHostGro
                 .updateAutoscaleConfigByClusterCrn(TEST_CLUSTER_CRN, distroXAutoscaleClusterRequest);
     }
 
+    @Test(expected = BadRequestException.class)
+    public void testUpdateAutoscaleConfigWithTimeAlertsAndStopStartEnabled() {
+        List<TimeAlertRequest> timeAlertRequests = getTimeAlertRequests(2, List.of("compute", "compute"));
+        DistroXAutoscaleClusterRequest distroXAutoscaleClusterRequest = new DistroXAutoscaleClusterRequest();
+        distroXAutoscaleClusterRequest.setTimeAlertRequests(timeAlertRequests);
+        distroXAutoscaleClusterRequest.setEnableAutoscaling(true);
+        distroXAutoscaleClusterRequest.setUseStopStartMechanism(true);
+
+        distroXAutoScaleClusterV1Endpoint.updateAutoscaleConfigByClusterName(TEST_CLUSTER_NAME, distroXAutoscaleClusterRequest);
+    }
+
+    @Test(expected = BadRequestException.class)
+    public void testEnableStopStartScalingWithPreExistingTimeAlerts() {
+        DistroXAutoscaleClusterRequest distroXAutoscaleClusterRequest = new DistroXAutoscaleClusterRequest();
```

Comment:
This is the test I was referring to earlier. I think distroXAutoscaleClusterRequest is valid even without any scaling policies defined (The CLI may be setting these, but doesn't necessarily have to)

Reply:
Added a separate test case to validate the mentioned scenario.